react-components/src/library/components/stem-finder-result.js

```diff
@@ -94,10 +94,18 @@ const StemFinderResult = Component({
     }.bind(this)
     if (resource.is_favorite) {
       jQuery.post('/api/v1/materials/remove_favorite', { favorite_id: resource.favorite_id }, done)
-      ga('send', 'event', 'Favorite Button', 'Click', `${resource.name} removed from favorites`)
+      gtag('event', 'click', {
+        'category': 'Favorite Button',
+        'action': 'opened',
```

Comment:
This `'action': 'opened',` doesn't seem to match up anything in the previous event tracking code. Did you mean to add it?